### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false positives for `base-select` of `appearance`

### ISSUE THREAD

**Issue by otomad** [stylelint/stylelint#8778]
### What minimal example or steps are needed to reproduce the bug?

```css
select {
	appearance: base-select;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
    "rules": {
        "declaration-property-value-no-unknown": true
    }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIDOMA2MwBcAEwA6AdnpgIYAOZMJATiQWDPNgEYkoC0K6WA3IQL4gANCABmAS3QA5EgFs4iGAA85ZdADowSJMPAQCEgOYIQ+AtgvY8IagFd0Sa0zOXXVkLDCoaJTOP3sZNQQlNSYAJ7sAG4kqLYw7AQQ7LYEANZJAO5ECNiYdjCErvwCupAG4oYAYhDUsr4mAFZI+rqwZDqILu5IEeio4gSYTu7emDC91kJFPX1og5js5UacpARQNFAj1mMTwyCl-EA)

### Which Stylelint-related dependencies are you using?

```json
{
	"stylelint": "16.24.0"
}
```

### What did you expect to happen?

No problems to be reported.

### What actually happened?

The following problems were reported:

```
example.css
2:14-25 error Unexpected unknown value "base-select" for property "appearance" (declaration-property-value-no-unknown)
```

### Do you have a proposal to fix the bug?

Update syntax in CSSTree.

For temporary use, you can extend the syntax yourself:
```json
{
	"languageOptions": {
		"syntax": {
			"properties": { 
				"appearance": "| base-select",
			}
		}
	}
}
```
